pwiz_tools/Skyline/Util/UtilUIExtra.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System;
-using System.Diagnostics;

```

Comment:
I don't see how the changes in this file are related. So, maybe they are just left over from originally putting the class in this file.

Reply:
Yeah, that's cruft, I will revert.